src/OpenDebugAD7/Telemetry/DebuggerTelemetry.cs

```diff
@@ -28,6 +28,7 @@ internal sealed class DebuggerTelemetry
         public const string TelemetryAttachEventName = "Attach";
         public const string TelemetryEvaluateEventName = "Evaluate";
         public const string TelemetryPauseEventName = "Pause";
+        public const string TelemetryTracepointEventName = "Tracepoint";
```

Comment:
You should add telemetry to the session telemetry to show how often tracepoints are used. 

Also add failure telemetry to see how often your exception gets thrown and add some information on why its failing to parse w/o adding the actual text.